pkg/compiler/limit_test.go

```diff
@@ -0,0 +1,23 @@
+package compiler_test
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"testing"
+)
+
+// Test for #605, #623.
+// Codegen should emit integers in proper format.
+func TestManyVariables(t *testing.T) {
+	const count = 155
```

Comment:
Why 155?

Reply:
Because it was in the contract where I have encountered this bug. It is necessary to have MSB set to 1. I have added a comment.